app/policies/proposal_policy/scope.rb

```diff
@@ -0,0 +1,20 @@
+class ProposalPolicy
+  class Scope
+    def initialize(user, scope)
+      @user = user
+      @scope = scope
+    end
+
+    # returns the entire list of Proposals that are visible to the user
+    def resolve
+      if @user.admin?
+        @scope.all
+      elsif @user.client_admin?
+        # TODO include all Proposals that the user is involved in
```

Comment:
This is the only (known) change in behavior: client admins won't be able to see their Proposals that they are involved with from other clients. This is a very narrow edge case, and it greatly simplifies the query to not worry about it.